velox/docs/develop/operators.rst

```diff
@@ -619,6 +619,8 @@ If no sorting columns are specified then the order of the results is unspecified
     - Output column names for each window function invocation in windowFunctions list below.
   * - windowFunctions
     - Window function calls with the frame clause. e.g row_number(), first_value(name) between range 10 preceding and current row. The default frame is between range unbounded preceding and current row.
+  * - inputsSorted
+    - If true, the Window operator assums that the inputs are clustered on partition keys and sorted on sorting keys in sorting orders. In this case, the operator splits the window partition and begin processing it as soon as it receives the data. This allows for parallel processing of the window partitions. If false, the Window operator accumulates all inputs first, then sorts the data, splits the window partition based on the defined criteria, and then process each window partition sequentially.
```

Comment:
typos: 

- assums -> assumes
- "begin processing" -> "begins processing" or "starts processing"
- "process each" -> "processes each"


> This allows for parallel processing of the window partitions.

I'm not sure I understand this sentence. Would you clarify what does "parallel processing" mean here?

Reply:
@mbasmanova `parallel processing` is inaccurate. Thanks for pointing it out for me. I have removed this sentence.